client/src/reducers/getMealReviewsReducer.js

```diff
@@ -0,0 +1,16 @@
+import {GET_MEAL_REVIEWS} from "../actions/actionTypes"
+
+const initialState = {};
+
+export default function (state = initialState,action) {
+  switch(action.type) {
+    case `${GET_MEAL_REVIEWS}_FULFILLED`:
+      return { data: action.payload.data, isFulfilled: true };
```

Comment:
We need to set isPending to false in fulfilled and rejected.

Reply:
@matthall8, will do some research on this

Reply:
Did some research on this with Mario. The initialState is reset every time this reducer is called so it turns out we don't need to worry about previous isFulfilled etc.
here is a snap of the result:
![image](https://user-images.githubusercontent.com/28222381/45599004-242b3800-b9ed-11e8-9091-b66841564c09.png)